src/encoded/static/components/app.js

```diff
@@ -385,7 +382,7 @@ export default class App extends React.Component {
 
     listActionsFor(category) {
         if (category === 'context') {
-            var context = this.currentResource();
```

Comment:
Removed this.currentResource() as it 'masks' the action of grabbing this.props.context more than clarifies it. And is not needed/used in general.